src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImpl.java

```diff
@@ -154,7 +154,6 @@ public RassXmlProcessingResults updateKFS(List<RassXmlFileParseResult> successfu
         
         return new RassBusinessObjectUpdateResultGrouping<R>(objectDefinition.getBusinessObjectClass(), objectResults, groupingResultCode);
     }
-
```

Comment:
This whitespace removal appears to be the only change to this particular file. Can this specific change be removed?

Reply:
I have removed this change. However, I don't think 2 newlines are needed here.